service-proxy.js

```diff
@@ -626,6 +636,61 @@ function isExitFor(serviceName) {
     return chan.serviceProxyMode === 'exit';
 };
 
+ServiceDispatchHandler.prototype.emitPeriodicStats =
+function emitPeriodicStats() {
+    var self = this;
+    self.periodicStatsHandle = null;
+
+    var serviceNames = Object.keys(self.channel.subChannels);
+    for (var index = 0; index < serviceNames.length; index++) {
+        var serviceName = serviceNames[index];
+        var serviceChannel = self.channels.subChannels[serviceName];
+        self.emitPeriodicServiceStats(serviceChannel, serviceName);
```

Comment:
pretty sure `serviceChannel.serviceName === serviceName`, could get away without the extra arg